mungegithub/mungers/mungers.go

```diff
@@ -88,6 +89,7 @@ func InitializeMungers(config *github.Config, features *features.Features) error
 		if err := munger.Initialize(config, features); err != nil {
 			return err
 		}
+		glog.Infof(mungerutil.PrettyString(munger))
```

Comment:
We were already using `PrettyMarshal` within the submit-queue munger, so, I just repurposed that to print a string instead. Would we gain anything by going with that package?